src/api/client.ts

```diff
@@ -82,6 +83,16 @@ export function getNostoClient(): PromiseLike<NostoClient> {
     })
 }
 
+export function logAndCaptureError(message: string, error: unknown, level: 'debug' | 'info' | 'warn' | 'error') {
```

Comment:
logAndCaptureError -> log ?
and maybe overload the function to have the variant without error argument available as well?

Reply:
I am not sure what you mean by overloading the function, I can add an object parameter and see if message exists in it as an optional one, but I'm not entirely sure about overloading, could you go into alittle bit more detail? 🙏